.travis.yml

```
@@ -51,6 +51,10 @@ jobs:
       script:
         - go mod vendor
         - RUN_SA_OPERATOR=TRUE make test PARALLEL_COUNT=2
+          # Run multinamespace tests as admin
+          # skip role settings check for now
+        - kubectl config use-context $TESTING_CONTEXT
+        - make multinamespace-test PARALLEL_COUNT=2
```

```suggestion
        - make multinamespace-test
```
PARALLEL_COUNT is not needed for this test, no t.Parallel() or t.Run(...) calls there. But can be keep as not adds additional problems